Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -23,6 +23,16 @@ public IQueryable<Candidate> GetByProject(Guid projectId)
             return GetCollection().AsQueryable().Where(x => x.ProjectResults.Any(y => y.ProjectId == projectId));
         }
 
+        public IQueryable<Candidate> GetCandidatesPassedTestByProject(Guid projectId)
+        {
+            return GetCollection().AsQueryable().Where(x => x.ProjectResults.Any(y => y.ProjectId == projectId && y.Status == Status.Test));
+        }
+
+        public IQueryable<Candidate> GetUnassignedCandidatesByProject(Guid projectId)
+        {
+            return GetCollection().AsQueryable().Where(x => x.ProjectResults.Any(p => p.ProjectId == projectId && !p.IsAssigned && (p.Status == Status.RecruiterInterview || p.Status == Status.TechInterviewOneStep || p.Status == Status.TechInterviewSecondStep)));
```

Comment:
rename IsAssigned to IsAssignedOnInterview, and in dto too

Reply:
+ scripts

Reply:
done